week-2/implement/vat.js

```diff
@@ -7,4 +7,14 @@
 
 // Given a number,
 // When I call this function with a number
-// Then it returns the new price with VAT added on
+// it returns the new price with VAT added on
+
+function priceWithVAT(price) {
+  newPrice = price * 1.2;
+  return `New price (20% VAT inclusive): £${newPrice}`;
```

Comment:
Ideally, this should just return a number called `newPrice` without any string/text before to it.